PacketDotNet/IPv6Fields.cs

```diff
@@ -88,7 +90,7 @@ public struct IPv6Fields
         /// <summary>
         /// The byte length of the IPv6 Header
         /// </summary>
-        public static readonly Int32 HeaderLength; // == 40
```

Comment:
Please undo this change, this will break existing stuff (and HeaderLength is similar to the rest).

Reply:
Still applies.

Reply:
Sorry, should be good now.